HelpMyStreetFE/HelpMyStreetFE/Controllers/RegistrationController.cs

```diff
@@ -204,9 +204,9 @@ public async Task<ActionResult> StepThreePost([FromForm] StepThreeFormModel form
 
                 await _groupMemberService.AddUserToDefaultGroups(user.ID);
 
-                if (user.ReferringGroupId.HasValue)
+                if (user.ReferringGroupId.HasValue && user.ReferringGroupId != -1)
```

Comment:
@gburnhill better to use an enum (Groups.Generic) or constant rather than a "magic number" wherever possible.